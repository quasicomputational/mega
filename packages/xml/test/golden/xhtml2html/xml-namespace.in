<html xml:space="preserve" xmlns="http://www.w3.org/1999/xhtml"><head /><body>Hello!</body></html>
