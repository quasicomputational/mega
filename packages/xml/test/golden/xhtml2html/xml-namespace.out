<!DOCTYPE html><html xml:space=preserve><head></head><body>Hello!</body></html>